fastlane/swift/Deliverfile.swift

```diff
@@ -11,4 +11,4 @@ class Deliverfile: DeliverfileProtocol {
 
 
 
-// Generated with fastlane 2.68.2
\ No newline at end of file
+// Generated with fastlane 2.69.2
```

Comment:
Unrelated of course, but could we add the missing new line to the last line of each file?

Reply:
For you? Anything.